gemrb/GUIScripts/pst/GUIINV.py

```diff
@@ -34,8 +34,6 @@
 ItemAmountWindow = None
 OverSlot = None
```

Comment:
can we please get rid of this global and just pass it directly to `UpdateSlot`?

or probably better yet move
```
if GemRB.CanUseItemType (SlotType["Type"], itemname):
			Button.SetState (IE_GUI_BUTTON_SELECTED)
		else:
			Button.SetState (IE_GUI_BUTTON_ENABLED)
```

from `UpdateSlot` directly to  `MouseEnterSlot`

AFICT, every other interaction with `OverSlot` becomes useless with that change and can just be deleted

Reply:
Actually, I can see this is done in `InventoryCommon`... I'll fix it there and you can probably just use the `InventoryCommon` code.

Reply:
Can you please pull in the updates from `InventoryCommon`. This global is undesireable.

Reply:
Did the latest push catch it? I think I had missed a step in the rebase attempt

Reply:
well I didn't change the PST version. I was hoping you would just use the `MouseEnter` and `MouseLeave` handlers from the common version. Anything that still has `OverSlot` needs to be killed.

Reply:
Ah of course, apologies, it's a bit late here, I misunderstood. 

That is what I will try to do next, there is just a bit more I need to straighten out before I can do that, for now please consider this as still WIP